src/pycolmap/estimators/pose.cc

```diff
@@ -181,4 +230,20 @@ void BindAbsolutePoseEstimator(py::module& m) {
         "followed by non-linear refinement.");
   DefDeprecation(
       m, "absolute_pose_estimation", "estimate_and_refine_absolute_pose");
+
+  m.def("estimate_relative_pose",
+        &PyEstimateRelativePose,
+        "points1"_a,
+        "points2"_a,
```

Comment:
Should we make it clear that the points are expected to be in normalized coordinates? e.g. `points1_in_cam` or so? Similarly for the C++ definitions and for the refinement function.

Reply:
Done.